httpcore5-h2/src/main/java/org/apache/hc/core5/http2/nio/pool/H2ConnPool.java

```diff
@@ -137,7 +137,7 @@ protected void validateSession(
             final IOSession ioSession,
             final Callback<Boolean> callback) {
         final TimeValue timeValue = validateAfterInactivity;
-        if (TimeValue.isPositive(timeValue)) {
+        if (TimeValue.ZERO_MILLISECONDS.equals(timeValue) || TimeValue.isPositive(timeValue)) {
```

Comment:
@pfrank13 Any reason for not using `TimeValue#compareTo` here?

Reply:
Sorry missed your message hopefully what I changed it works